tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -368,12 +390,27 @@ def mirror_status(self, *args):
 
     # Add Peer
     def peer_add(self, **kw):
-        return self.exec_cmd(
-            cmd="rbd mirror pool peer add {} {}".format(
-                kw.get("poolname"), kw.get("cluster_spec")
+            return self.exec_cmd(
+                    cmd=f"rbd mirror pool peer add {kw['poolname']} {kw['cluster_spec']"
             )
+    
+    #Create bootstrap token on primary cluster
+    def bootstrap_peers(self, **kw):
+        return self.exec_cmd(
+                cmd=f"rbd mirror pool peer bootstrap create --site-name master {kw['poolname']} > /root/bootstrap_token_master"
         )
 
+    #Import bootstrap token from scecondary cluster
+    def import_bootstrap(self, **kw):
```

Comment:
Instead of a comment, it would be nice if you could add docstrings to the method like
```
def import_bootstrap(self, **kw):
  """Return the result of token import in the secondary cluster."""
```